iep-rxhttp/src/main/java/com/netflix/iep/http/ServerEntry.java

```diff
@@ -52,6 +52,9 @@ long ctime() {
   List<Server> next(int n) {
     List<Server> out = new ArrayList<>(n);
     int size = servers.size();
+    if (size == 0) {
+      return new ArrayList<>();
+    }
```

Comment:
I think it would be better to move it to the beginning of the function and do something like:

```java
if (servers.isEmpty()) {
  return Collections.emptyList();
}
```